README.adoc

```diff
@@ -337,17 +503,53 @@ inventory-deployment-cf8f564c6-nctcr   1/1       Running   0          8m
 
 == Testing the microservices
 
+// Static guide instruction
+ifndef::cloud-hosted[]
 Run the tests by running the following command and appropriately substituting `[hostname]` 
 for the correct value.
 
 [role=command]
 ```
 mvn failsafe:integration-test -Dcluster.ip=[hostname]
 ```
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Go back to the terminal where you set up the port forwarding for the second pod.
+Press **CTRL+C** to stop the port forwarding.
+Run the following commands to set up port forwarding to access the **system** service.
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+kubectl port-forward svc/system-service $SYSTEM_NODEPORT:9080
+```
+{: codeblock}
+
+Update the **pom.xml** files so that the **system.service.root** and **inventory.service.root** properties
+match the values to access the **system** and **inventory** services.
+
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+INVENTORY_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services inventory-service`
+sed -i 's=localhost:31000='"localhost:$SYSTEM_NODEPORT"'=g' inventory/pom.xml
+sed -i 's=localhost:32000='"localhost:$INVENTORY_NODEPORT"'=g' inventory/pom.xml
+sed -i 's=localhost:31000='"localhost:$SYSTEM_NODEPORT"'=g' system/pom.xml
+```
+{: codeblock}
+
+Run the integration tests by using the following command:
+
+```
+mvn failsafe:integration-test \
+    -Dsystem.service.root=localhost:$SYSTEM_NODEPORT \
+    -Dinventory.service.root=localhost:$INVENTORY_NODEPORT
+```
+{: codeblock}
+endif::[]
 
-A few tests are included for you to test the basic functions of the microservices. If a test failure
-occurs, then you might have introduced a bug into the code. To run the tests, wait for all pods to be
-in the ready state before proceeding further.
+A few tests are included for you to test the basic functions of the microservices.
+If a test fails, then you might have introduced a bug into the code.
+To run the tests, wait for all pods to be in the ready state before proceeding further.
```

Comment:
To run the tests, wait for all pods to be in the ready state before proceeding further.
--->
Wait for all pods to be in the ready state before you run the tests.